pkg/compat/client.go

```diff
@@ -219,3 +222,31 @@ func (c client) Update(ctx context.Context, in runtime.Object) error {
 	}
 	return c.Client.Update(ctx, obj)
 }
+
+//
+// Parses gitVersion ("v1.11.0+d4cacc0"), returns major, minor, err
+func parseGitVersion(gitVersion string) (int, int, error) {
```

Comment:
Recommend making this a _method_ instead of a package function unless you indent for it to be generally used within the whole package.